src/app/shared/components/header/header.component.html

```diff
@@ -1,16 +1,13 @@
 <header class="bg-primary">
-  <nav
-    class="mx-auto flex max-w-7xl items-center justify-between p-6 lg:px-8"
-    aria-label="Global"
-  >
+  <link rel="icon" href="/assets/favicon/favicon.ico" sizes="32x32">
+  <link rel="icon" href="/assets/favicon/hf_logo.svg" type="image/svg+xml">
+  <link rel="apple-touch-icon" href="/assets/favicon/apple-touch-icon.png"><!-- 180×180 -->
+  <link rel="manifest" href="/assets/favicon/manifest.webmanifest">
+  <nav class="mx-auto flex max-w-7xl items-center justify-between p-6 lg:px-8" aria-label="Global">
```

Comment:
header != head

Please use the `index.html` file. This component renders a header element in the html body.

Reply:
moved in 566d0ad454b8184a3b2c4259937eda2c9444f536

Reply:
nope still there

Reply:
you copied it to the index.html, but forgot to remove it here

Reply:
I removed it, but I got some issues with git.
But now it should be gone ^^

d59dd546d87a91c85b706de9e834f21e20eb0cca